webapp/src/Form/Type/RejudgingType.php

```diff
@@ -44,6 +44,16 @@ public function __construct(EntityManagerInterface $em)
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder->add('reason', TextType::class);
+        $builder->add('priority', ChoiceType::class,
+            [
+                'choices' => [
+                    'low' => 'low',
```

Comment:
Why not set the keys (or values, I always forget) to the actual prio numbers? Symfony will validate you actually pass a valid value so you can’t “hack” the form and specify something else (well technically because we have the Ajax in between you could but 🤷‍♂️)

Reply:
You could also use `curl` or `httpie`, and I don't want to accept arbitrary priorities.

Reply:
Fair point